packages/@ember/-internals/views/lib/system/event_dispatcher.js

```diff
@@ -285,13 +285,16 @@ export default EmberObject.extend({
           return;
         }
 
+        let result = true;
         for (let index = 0; index < actions.length; index++) {
           let action = actions[index];
 
           if (action && action.eventName === eventName) {
-            return action.handler(event);
+            // return false if any of the action handlers returns false
+            result = result && action.handler(event);
```

Comment:
I think this is not _quite_ right: if the first handler returned false, that will prevent the remaining handlers (on the same elements) from running. That would be `event.stopImmediatePropagation()` semantics in jQuery, and I don't think that's what we want here.

Reply:
Ah yes, excellent catch! I switched the order of those `&&` operands, and added another test to cover this!